base/errorshow.jl

```diff
@@ -644,33 +814,20 @@ function show_backtrace(io::IO, t::Vector)
         end
     end
 
-    print(io, "\nStacktrace:")
-    if length(filtered) < BIG_STACKTRACE_SIZE
-        # Fast track: no duplicate stack frame detection.
-        try invokelatest(update_stackframes_callback[], filtered) catch end
-        frame_counter = 0
-        for (last_frame, n) in filtered
-            frame_counter += 1
-            show_trace_entry(IOContext(io, :backtrace => true), last_frame, n, prefix = string(" [", frame_counter, "] "))
-        end
+    if length(filtered) > BIG_STACKTRACE_SIZE
+        show_reduced_backtrace(IOContext(io, :backtrace => true), filtered)
         return
     end
 
-    show_reduced_backtrace(IOContext(io, :backtrace => true), filtered, true)
-end
+    println(io, "\nStacktrace:")
 
-function show_backtrace(io::IO, t::Vector{Any})
-    # t is a pre-processed backtrace (ref #12856)
-    if length(t) < BIG_STACKTRACE_SIZE
-        try invokelatest(update_stackframes_callback[], t) catch end
-        for entry in t
-            show_trace_entry(io, entry...)
-        end
-    else
-        show_reduced_backtrace(io, t, false)
-    end
+    # process_backtrace returns a Tuple{Frame, Int}
+    frames = first.(filtered)
```

Comment:
This variable isn't used; did you mean to pass it instead of `t` in the next call?